includes/orders/classes/class-order.php

```diff
@@ -444,6 +445,35 @@ public function get_fees() {
 		return $fees;
 	}
 
+	/**
+	 * Retrieve the credits applied to the order.
+	 * These exist only for manually added orders.
+	 *
+	 * @since 3.0
+	 *
+	 *@return Order_Adjustment[] Order credits.
+	 */
+	public function get_credits() {
+		// Default values
+		$credits = array();
+
+		// Ensure adjustments exist.
+		if ( null === $this->adjustments ) {
+			$this->adjustments = $this->get_adjustments();
+		}
+
+		// Fetch the fees that applied to the entire order.
+		foreach ( $this->adjustments as $adjustment ) {
```

Comment:
This could just be refactored to do:

```suggestion
		foreach ( $this->get_adjustments() as $adjustment ) {
```

And remove the above `null` check. Might be a bit less repetitive.